stable/spark/values.yaml

```diff
@@ -5,8 +5,8 @@
 
 Master:
   Name: master
-  Image: "k8s.gcr.io/spark"
-  ImageTag: "1.5.1_v3"
+  Image: "p7hb/docker-spark"
```

Comment:
/cc @lachie83

Reply:
Why aren't they using the official source repository in the Dockerfile? https://github.com/P7h/docker-spark/blob/2.2.0/Dockerfile#L18